### PR TITLE
Small changes and comments for the first tools

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: tools/astropytools/fits2bitmap.xml [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-        fits2bitmap '$input_fits' --stretch '$stretch' --cmap '$cmap' -o out.png; mv -fv out.png '$output_png'
+        fits2bitmap
+        '$input_fits'
+        --stretch '$stretch'
+        --cmap '$cmap'
+        -o out.png
     ]]></command>
     <inputs>
         <param type="data" name="input_fits" format="fits" />
         <param type="text" name="stretch" />
```
Comment (bgruening) [inline tools/astropytools/fits2bitmap.xml:15]: labels and help is missing for all parameters

**Review by bgruening — COMMENTED** [review]
fits2bitmap should have a better tool help the current one is not much useful

File: tools/astropytools/fitsinfo.xml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
     <requirements>
         <requirement type="package" version="5.1">astropy</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
-        fitsinfo '$input1' | grep -vi filename > '$output1' 
+        fitsinfo '$input' | grep -vi filename > '$output' 
     ]]></command>
     <inputs>
-        <param type="data" name="input1" format="fits" />
+        <param type="data" name="input" format="fits" />
```
Comment (bgruening) [inline tools/astropytools/fitsinfo.xml:9]: label and help could be added and the overall tool help could be improved.

Its not very useful to explain the parameters, for that you have label and help on the params. Instead write what the tool is doing, explain the output etc ...

**Comment by bgruening** [comment]
@martenson could you please add the category "Astronomy" to the TS? 

Thanks!

**Review by volodymyrss — APPROVED** [review]

**Comment by volodymyrss** [comment]
Ok, thanks!
I'll just merge your suggestions here and make the labels and help in a separate PR following an [issue](https://github.com/esg-epfl-apc/tools-astro/issues/6)

**Comment by bgruening** [comment]
Great! Thanks a lot!

**Comment by martenson** [comment]
> @martenson could you please add the category "Astronomy" to the TS?
> 
> Thanks!

done

**Pull request merged**